components/region_cache_memory_engine/src/background.rs

```diff
@@ -37,11 +43,97 @@ fn parse_write(value: &[u8]) -> Result<WriteRef<'_>, String> {
     }
 }
 
+pub struct BackgroundWork {
```

Comment:
```suggestion
pub struct BackgroundWorker {
```

Reply:
And it's better to have a unit test for this struct too

Reply:
> And it's better to have a unit test for this struct too

It's just manages the worker start/stop and task schedule. `test_gc_worker` tests the worker.